framework/wazuh/cluster/common.py

```diff
@@ -234,27 +244,23 @@ def get_messages(self) -> Tuple[bytes, int, bytes]:
                     decrypted_payload = self.my_fernet.decrypt(bytes(self.in_msg.payload)) if self.my_fernet is not None \
                                                                                            else bytes(self.in_msg.payload)
                 except cryptography.fernet.InvalidToken:
-                    self.logger.error("Could not decrypt message. Check key is correct.")
-                    decrypted_payload = b'Could not decrypt message. Check key is correct.'
-                    self.in_msg.cmd = b'err'
+                    raise exception.WazuhClusterError(3025)
                 yield self.in_msg.cmd, self.in_msg.counter, decrypted_payload
                 self.in_msg = InBuffer()
             else:
                 break
             parsed = self.msg_parse()
 
-    async def send_request(self, command: bytes, data: bytes, error_cmd: bytes = b'err') -> bytes:
+    async def send_request(self, command: bytes, data: bytes) -> bytes:
         """
         Sends a request to peer
 
         :param command: command to send
         :param data: data to send
-        :param error_cmd: Cmd to send in case of error
         :return: response from peer.
         """
         if len(data) > self.request_chunk:
-            self.logger.error("Error sending request '{}': Max payload length exceeded".format(command))
-            command, data = error_cmd, b"Error: Command '" + command + b"' Max msg length exceeded."
+            raise exception.WazuhClusterError(f"Error sending request '{command}': Max payload length exceeded")
```

Comment:
This line probably will fail because an error code is expected in the first argument of WazuhClusterError